frontend/tests/unit/components/inputs/Input.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { mount } from '@vue/test-utils';
+
+import Input from '@/components/inputs/Input.vue';
+
+const createWrapper = (props = {}, attachToBody = false) => {
```

Comment:
For consistency, I rather would like to see a typed options object as a parameter for this function. Just as the `createWrapper` of the `TextInput` tests looked like with an additional `type` option. So everything should be in the options object, also the `attachToBody` parameter. Most important is that we had typings for the options' properties.

Reply:
Thanks for reminding me that this pattern is the preferred one for our tests. Kinda missed it :see_no_evil:.
Will keep that in mind next time i am writing tests!